packages/jest-types/src/Config.ts

```diff
@@ -411,7 +411,7 @@ export type GlobalConfig = {
   passWithNoTests: boolean;
   projects: Array<string>;
   replname?: string;
-  reporters?: Array<string | ReporterConfig>;
```

Comment:
This should type normalized config, right? I have added tests `jest-config`, which show that `reporters` are always normalized to a tuple. Having more precise type here, makes it possible to have less logic inside `TestScheduler`. Last commits are added just to illustrate the solution. Perhaps it would be better to send separate PR? Feel free to revert them.